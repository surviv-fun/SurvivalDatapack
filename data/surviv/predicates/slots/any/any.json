[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "chest": {
                            "items": ["minecraft:elytra"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": [
                                "minecraft:netherite_sword",
                                "minecraft:diamond_sword",
                                "minecraft:golden_sword",
                                "minecraft:iron_sword",
                                "minecraft:stone_sword",
                                "minecraft:wooden_sword",

                                "minecraft:netherite_axe",
                                "minecraft:diamond_axe",
                                "minecraft:golden_axe",
                                "minecraft:iron_axe",
                                "minecraft:stone_axe",
                                "minecraft:wooden_axe",

                                "minecraft:netherite_pickaxe",
                                "minecraft:diamond_pickaxe",
                                "minecraft:golden_pickaxe",
                                "minecraft:iron_pickaxe",
                                "minecraft:stone_pickaxe",
                                "minecraft:wooden_pickaxe",

                                "minecraft:netherite_shovel",
                                "minecraft:diamond_shovel",
                                "minecraft:golden_shovel",
                                "minecraft:iron_shovel",
                                "minecraft:stone_shovel",
                                "minecraft:wooden_shovel",

                                "minecraft:netherite_hoe",
                                "minecraft:diamond_hoe",
                                "minecraft:golden_hoe",
                                "minecraft:iron_hoe",
                                "minecraft:stone_hoe",
                                "minecraft:wooden_hoe"
                            ]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": [
                                "minecraft:netherite_sword",
                                "minecraft:diamond_sword",
                                "minecraft:golden_sword",
                                "minecraft:iron_sword",
                                "minecraft:stone_sword",
                                "minecraft:wooden_sword",

                                "minecraft:netherite_axe",
                                "minecraft:diamond_axe",
                                "minecraft:golden_axe",
                                "minecraft:iron_axe",
                                "minecraft:stone_axe",
                                "minecraft:wooden_axe",

                                "minecraft:netherite_pickaxe",
                                "minecraft:diamond_pickaxe",
                                "minecraft:golden_pickaxe",
                                "minecraft:iron_pickaxe",
                                "minecraft:stone_pickaxe",
                                "minecraft:wooden_pickaxe",

                                "minecraft:netherite_shovel",
                                "minecraft:diamond_shovel",
                                "minecraft:golden_shovel",
                                "minecraft:iron_shovel",
                                "minecraft:stone_shovel",
                                "minecraft:wooden_shovel",

                                "minecraft:netherite_hoe",
                                "minecraft:diamond_hoe",
                                "minecraft:golden_hoe",
                                "minecraft:iron_hoe",
                                "minecraft:stone_hoe",
                                "minecraft:wooden_hoe"
                            ]
                        }
                    }
                }
            }
        ]
    }
]
