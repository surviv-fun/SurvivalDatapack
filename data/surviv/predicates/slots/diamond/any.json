[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": ["minecraft:diamond_sword", "minecraft:diamond_axe", "minecraft:diamond_pickaxe", "minecraft:diamond_shovel", "minecraft:diamond_hoe"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": ["minecraft:diamond_sword", "minecraft:diamond_axe", "minecraft:diamond_pickaxe", "minecraft:diamond_shovel", "minecraft:diamond_hoe"]
                        }
                    }
                }
            }
        ]
    }
]
