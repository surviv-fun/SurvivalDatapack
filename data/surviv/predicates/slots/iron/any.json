[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "head": {
                            "items": ["minecraft:turtle_helmet"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": ["minecraft:iron_sword", "minecraft:iron_axe", "minecraft:iron_pickaxe", "minecraft:iron_shovel", "minecraft:iron_hoe"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": ["minecraft:iron_sword", "minecraft:iron_axe", "minecraft:iron_pickaxe", "minecraft:iron_shovel", "minecraft:iron_hoe"]
                        }
                    }
                }
            }
        ]
    }
]
