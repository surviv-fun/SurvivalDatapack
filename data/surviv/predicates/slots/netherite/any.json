[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": ["minecraft:netherite_sword", "minecraft:netherite_axe", "minecraft:netherite_pickaxe", "minecraft:netherite_shovel", "minecraft:netherite_hoe"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": ["minecraft:netherite_sword", "minecraft:netherite_axe", "minecraft:netherite_pickaxe", "minecraft:netherite_shovel", "minecraft:netherite_hoe"]
                        }
                    }
                }
            }
        ]
    }
]
