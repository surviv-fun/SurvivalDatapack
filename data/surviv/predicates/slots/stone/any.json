[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": ["minecraft:stone_sword", "minecraft:stone_axe", "minecraft:stone_pickaxe", "minecraft:stone_shovel", "minecraft:stone_hoe"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": ["minecraft:stone_sword", "minecraft:stone_axe", "minecraft:stone_pickaxe", "minecraft:stone_shovel", "minecraft:stone_hoe"]
                        }
                    }
                }
            }
        ]
    }
]
