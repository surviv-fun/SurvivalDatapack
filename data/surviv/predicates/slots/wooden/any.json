[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": ["minecraft:wooden_sword", "minecraft:wooden_axe", "minecraft:wooden_pickaxe", "minecraft:wooden_shovel", "minecraft:wooden_hoe"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": ["minecraft:wooden_sword", "minecraft:wooden_axe", "minecraft:wooden_pickaxe", "minecraft:wooden_shovel", "minecraft:wooden_hoe"]
                        }
                    }
                }
            }
        ]
    }
]
