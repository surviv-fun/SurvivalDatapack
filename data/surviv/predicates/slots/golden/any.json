[
    {
        "condition": "minecraft:alternative",
        "terms": [
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "mainhand": {
                            "items": ["minecraft:golden_sword", "minecraft:golden_axe", "minecraft:golden_pickaxe", "minecraft:golden_shovel", "minecraft:golden_hoe"]
                        }
                    }
                }
            },
            {
                "condition": "minecraft:entity_properties",
                "entity": "this",
                "predicate": {
                    "equipment": {
                        "offhand": {
                            "items": ["minecraft:golden_sword", "minecraft:golden_axe", "minecraft:golden_pickaxe", "minecraft:golden_shovel", "minecraft:golden_hoe"]
                        }
                    }
                }
            }
        ]
    }
]
